crossfire/__init__.py

```diff
@@ -48,9 +48,10 @@ def flatten(data, nested_columns=None):
     nested_columns = set(nested_columns or NESTED_COLUMNS)
     if not nested_columns.issubset(NESTED_COLUMNS):
         raise NestedColumnError(nested_columns)
-    if isinstance(data, list):
-        keys = set(data[0].keys()) & nested_columns
-        for item in data:
-            for key in keys:
-                for k, v in item.get(key).items():
-                    return f"{key}_{k}", v
+    if isinstance(data, dict):
```

Comment:
Now, the function validate if data is a dict.
And I realized that it was returning each key and value. So the test still not passing. I added a `flattened_dict` to recieve the flattened columns an then return it...